eclair-core/src/test/scala/fr/acinq/eclair/channel/states/h/ClosingStateSpec.scala

```diff
@@ -1475,12 +1481,17 @@ class ClosingStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with
 
     // bob claims multiple htlc outputs in a single transaction (this is possible with anchor outputs because signatures
     // use sighash_single | sighash_anyonecanpay)
-    val bobHtlcTxs = bobRevokedTxs.htlcTxsAndSigs.collect {
-      case HtlcTxAndSigs(txInfo: HtlcSuccessTx, localSig, remoteSig) =>
+    val bobKeyManager = bob.underlyingActor.nodeParams.channelKeyManager
+    val bobChannelKeyPath = bobKeyManager.keyPath(bob.stateData.asInstanceOf[DATA_NORMAL].commitments.localParams, ChannelVersion.ANCHOR_OUTPUTS)
+    val bobPerCommitmentPoint = bobKeyManager.commitmentPoint(bobChannelKeyPath, bobRevokedCommit.index.toInt)
+    val bobHtlcTxs = bobRevokedCommit.htlcTxsAndRemoteSigs.collect {
+      case HtlcTxAndRemoteSig(txInfo: HtlcSuccessTx, remoteSig) =>
         val preimage = revokedCloseFixture.htlcsAlice.collectFirst { case (add, preimage) if add.id == txInfo.htlcId => preimage }.get
         assert(Crypto.sha256(preimage) === txInfo.paymentHash)
+        val localSig = bobKeyManager.sign(txInfo, bobKeyManager.htlcPoint(bobChannelKeyPath), bobPerCommitmentPoint, TxOwner.Local, commitmentFormat)
         Transactions.addSigs(txInfo, localSig, remoteSig, preimage, AnchorOutputsCommitmentFormat)
-      case HtlcTxAndSigs(txInfo: HtlcTimeoutTx, localSig, remoteSig) =>
+      case HtlcTxAndRemoteSig(txInfo: HtlcTimeoutTx, remoteSig) =>
+        val localSig = bobKeyManager.sign(txInfo, bobKeyManager.htlcPoint(bobChannelKeyPath), bobPerCommitmentPoint, TxOwner.Local, commitmentFormat)
```

Comment:
Same here, you don't need to actually sign, you can simply put `ByteVector64.Zeroes` as signatures.